ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml

```diff
@@ -40,6 +40,26 @@
     jobs:
         - 'ceph-ansible-prs-auto'
 
+- project:
+    name: ceph-ansible-prs-common-trigger
+    slave_labels: 'vagrant && libvirt && (smithi || centos7)'
```

Comment:
in fact, I believe we could change this to `vagrant && libvirt && centos7` but probably a dedicated commit/pr for that would make sense.